ddh-intro.md

```diff
@@ -2,31 +2,34 @@
 
 ![](http://docs.duckduckhack.com/assets/hack_search_engine.png)
 
-**We're a community dedicated to making DuckDuckGo the best search engine for programmers. We leverage 3rd party APIs, scrape external data sets, and write well-placed code to create what we call *Instant Answers*.**
+**We're a community dedicated to making DuckDuckGo the best search engine for programmers, and you can help!**
 
-You can think of [Instant Answers](https://duck.co/ia?topic=programming) as "perfect search results." They resolve queries above any ads or regular search results. Millions of searches on DuckDuckGo trigger Instant Answers, like [MDN Javascript](https://duckduckgo.com/?q=javascript+array&ia=about) and [Regex Cheat Sheet](https://duckduckgo.com/?q=regex+cheat+sheet&ia=cheatsheet&iax=1). 
+Our goal is to have a high quality, relevant Instant Answer for every search.
+We leverage 3rd party APIs and scrape external data sets to create these Instant Answers.
 
-Our Instant Answers are created and maintained by a community of open source developers around the world. Just like the Wikipedia community creates articles for topics – the DuckDuckHack community creates Instant Answers for searches! 
+## Programming Mission
 
-## Get Started
+The Programming Mission is the current focus of DuckDuckHack: improving search for the top 15 programming languages.
+For now, we are **only accepting Pull Requests and Issues related to the Programming Mission.**
 
-1. Start by learning about how DuckDuckHack is [Improving Programming Search Results]({{ book.forumURL }}/c/programming). All of our current open source projects begin here.
-2. You'll want to join the [DuckDuckHack Forum]({{ book.forumURL }}) to connect and collaborate with the community. There you'll find a current project to join and start hacking. 
-3. Use these docs and the community's support to help bring your own Instant Answer to life.
-4. Sign up for our [dev list](https://www.listbox.com/subscribe/?list_id=197814) to get the latest DuckDuckHack news, contribution suggestions and programming tips.
-5. Become a leader and all-star contributor helping to change the world of search through community!
+#### How to contribute
 
-[![slack](http://docs.duckduckhack.com/assets/slack.png) Ask questions on Slack]({{ book.slackURL }}) or [email us](mailto:{{ book.email }}) directly.
+See how to [Get Started](https://forum.duckduckhack.com/t/duckduckhack-getting-started/53) and:
 
-## Create a New Instant Answer
+- Choose a task from the Open Tasks table, or from GitHub:
```

Comment:
Yeah we know how to put anchors, see e.g. the [How Tos](https://forum.duckduckhack.com/t/howtos/959)